Firebase/Messaging/FIRMessaging.m

```diff
@@ -162,32 +162,19 @@ @interface FIRMessaging () <FIRMessagingInstanceProvider, FIRLibrary>
 
 @implementation FIRMessaging
 
-// File static to support InstanceID tests that call [FIRMessaging messaging] after
-// [FIRMessaging messagingForTests].
-static FIRMessaging *sMessaging;
-
 + (FIRMessaging *)messaging {
-  if (sMessaging != nil) {
-    return sMessaging;
-  }
   FIRApp *defaultApp = [FIRApp defaultApp];  // Missing configure will be logged here.
```

Comment:
Should all these logic inside dispatch_once? Or it no longer work like that?

Reply:
I see static variable can't be init more than once and since this no longer works for dispatch_once so we should not make it static.